dir/path.go

```diff
@@ -52,14 +67,19 @@ func checkError(err error) {
 	}
 }
 
-// Config returns the path of config.json
+// Config returns the ready-only path of config.json
 func (p *PathManager) Config() string {
 	path, err := p.ConfigFS.GetPath(ConfigFile)
 	checkError(err)
 	return path
 }
 
-// LocalKey returns path of the local private key and it's certificate
+// ConfigForWrite returns the writable path of config.json
+func (p *PathManager) ConfigForWrite(writeLevel WriteLevel) string {
```

Comment:
Instead of creating a separate function for reading and writing, can we write a function that takes UserLevel/SystemLevel as a param and determines the path internally?

Also, applicable to other similar functions.

Reply:
Sure, updated.
Also, renamed `WriteLevel` to `DirLevel`, and added `UnionLevel`